data/products/sle-micro/config.yaml

```diff
@@ -1,17 +1,22 @@
 config:
   scripts:
-    sle-micro:
+    sle-micro-scripts:
       - chost-strip-image
       - microos-add-variant
       - snapper-microos-config
       - zypp-disable-multiversion
       - zypp-set-excludedocs
       - zypp-set-onlyrequires
   services:
-    sle-micro:
+    sle-micro-services:
       - afterburn-checkin
       - afterburn-firstboot-checkin
       - afterburn-sshkeys@
       - afterburn-sshkeys.target
       - boot.device-mapper
       - docker
+  sysconfig:
+    sle-micro-sysconig:
+      - file: /etc/sysconfig/network/config
+        name: WICKED_DEBUG
```

Comment:
SLe Micro 5.3 drops wicked and moves to NetworkManager, this will need to be version specific.